airbyte-integrations/connectors/source-bigquery/source_bigquery/source.py

```diff
@@ -56,6 +57,12 @@
 # Source
 class SourceBigquery(ConcurrentSourceAdapter):
     logger = logging.getLogger("airbyte")
+    logger.level = {
```

Comment:
We already have some code allowing to change the log level (although less precise that what you wrote) (see [this](https://github.com/airbytehq/airbyte/blob/46b2188fe05d149adf4be4a2378363ee07aa95ff/airbyte-cdk/python/airbyte_cdk/entrypoint.py#L90-L93)). I don't mind having both but I would like to make sure they don't clash

Reply:
Yes but unfortunately the command is not run with `--debug` anymore in the [CDK 1.3.3](https://github.com/airbytehq/airbyte/blob/46b2188fe05d149adf4be4a2378363ee07aa95ff/airbyte-cdk/python/airbyte_cdk/test/entrypoint_wrapper.py#L166-L189).